eng/common/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -17,17 +16,20 @@ steps:
   workingDirectory: $(Build.BinariesDirectory)
   displayName: Download and Extract azcopy Zip
 
-- task: Powershell@2
+- task: AzurePowerShell@5
+  displayName: 'Copy Docs to Blob Storage'
+  continueOnError: false
   inputs:
-    filePath: ${{ parameters.ScriptPath }}
-    arguments: >
+    azureSubscription: 'Azure SDK Artifacts'
+    ScriptType: 'FilePath'
+    ScriptPath: ${{ parameters.ScriptPath }}
+    ScriptArguments: >
       -AzCopy $(Resolve-Path "$(Build.BinariesDirectory)/azcopy/azcopy_windows_amd64_*/azcopy.exe")[0]
```

Comment:
Orthogonal but we should probably also remove this azcopy approach given it is already on all the agent machines.